lib/ansible/modules/user.py

```diff
@@ -86,12 +86,13 @@
         version_added: "2.0"
     password:
         description:
-            - Optionally set the user's password to this encrypted value.
+            - Optionally set the user's password hash to this value.
+            - Do B(not) put the actual password into this field.
```

Comment:
This may cause confusion with the statement lower, that says that you are supposed to do this for macOS.  So perhaps this statement and the `macOS` statement should be combined.

Reply:
I was confused by the macOS sentence too.
So on macOS the password field expects the actual password, on Linux the hash?
What is expected on Windows?

Reply:
to use win_user instead

Reply:
I gave the phrasing another try.

Regarding the differing behavior of Linux and macOS: I wonder if I should open an issue to discuss this behavior. Using the actual password on one platform and the hash on another one sounds like a foot gun to me.